src/core/analogsensor.cpp

```diff
@@ -807,9 +808,12 @@ void AnalogSensor::publish_values(const bool force) {
             std::string topic_str(topic);
             config["def_ent_id"] = topic_str.substr(0, topic_str.find("/")) + "." + uniq_s;
 
+            // dev section with model is only created on the 1st sensor
+            Mqtt::add_ha_dev_section(config.as<JsonObject>(), "Analog Sensors", nullptr, "EMS-ESP", EMSESP_APP_VERSION, !ha_dev_created);
```

Comment:
`add_dev_section` is already called 6 lines on top of this addition. Both needed?